polylogue/services/attachment_text.py

```
@@ -0,0 +1,182 @@
+        if suffix in {".html", ".htm"} or (mime and mime.startswith("text/html")):
+            text = _strip_html(text)
+        return AttachmentText(text=text, mime=mime, truncated=truncated, ocr_used=False, size_bytes=size)
+
+    # JSON and YAML can be normalised to reduce noise
+    if suffix in {".json", ".yaml", ".yml"}:
+        text, truncated = _read_text_with_limits(path, max_bytes=max_bytes, max_chars=max_chars)
+        try:
+            parsed = json.loads(text)
+            text = json.dumps(parsed, indent=2, ensure_ascii=False)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)